src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -98,6 +98,12 @@ def show_webapp(resource_group_name, name, slot=None, app_instance=None):
     return webapp
 
 
+def update_webapp(instance, client_affinity_enabled=None):
+    if client_affinity_enabled is not None:
+        instance.client_affinity_enabled = client_affinity_enabled == 'true'
```

Comment:
Actually, simply this statement is enough. You don't need to test the condition since the `None == 'true'` is always `False`

Reply:
We only want to change the field if it is given though. If it isn't provided, this will always change it to false.

Reply:
Ok.